static/flow/views/pi-selector-panel.js

```diff
@@ -516,13 +516,15 @@ var PiSelectorPanel = function(options) {
         //
         // Set name on program
         //
-        var name = jQuery('#program-editor-filename').val();
+        var displayedName = jQuery('#program-editor-filename').val();
         //var name = editor.getFileManager().getProgramName();
-        programSpec.name = name;
+        programSpec.name = ""; //don't have an actual folder location for this "virtual" program
```

Comment:
OK I see, no point in storing the location of the program, since it's all getting copied anyway